Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
```

Comment:
### [autoprefixer-rails](https://github.com/ai/autoprefixer-rails)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 9.4.8 → 9.8.6.5

[_[change-log](https://github.com/ai/autoprefixer-rails/blob/master/CHANGELOG.md), [source-code](https://github.com/ai/autoprefixer-rails), [gem-diff](https://my.diffend.io/gems/autoprefixer-rails/9.4.8/9.8.6.5)_]





<details>
<summary>Commits</summary>

A change of **106** commits. See the full changes on [the compare page](https://github.com/ai/autoprefixer-rails/compare/9.4.8...9.8.6.5).

These are the first 10 commits:
- (197a94a) [Update autoprefixer.js with Grid fix](https://github.com/ai/autoprefixer-rails/commit/197a94a04d4dabf4c6b80f33b0494a91d6d707d4)
- (b6c1941) [Release 9.4.9 version](https://github.com/ai/autoprefixer-rails/commit/b6c194149ccff72be36d458481bac7a25812984e)
- (7d389d1) [Add Rubygems links (#152)](https://github.com/ai/autoprefixer-rails/commit/7d389d167e7facbd78ae1c8a762442f5927a6407)
- (df16bca) [Give StandardRB a shot (#153)](https://github.com/ai/autoprefixer-rails/commit/df16bca9eb3a4633a92997e70dd8498b11e1be53)
- (dfcb2d5) [Clean up Compass config](https://github.com/ai/autoprefixer-rails/commit/dfcb2d567d039bf6c12bc031e7e1f9f2154509c2)
- (9d8cec1) [Update autoprefixer.js with warning for named Grid lines](https://github.com/ai/autoprefixer-rails/commit/9d8cec1a41187bc0d530f611b65c0c644ca4a7f8)
- (4831821) [Release 9.4.10 version](https://github.com/ai/autoprefixer-rails/commit/4831821dff296ee27a8b75256d3aabbdaea828b1)
- (f5facc2) [Fix NameError when processing without Rails (#154)](https://github.com/ai/autoprefixer-rails/commit/f5facc23699752df6071c203790808507f3540f3)
- (e19717c) [Update auroprefixer.js with fresh Can I Use](https://github.com/ai/autoprefixer-rails/commit/e19717cc9f9582aeb94508c3367660ea684de59f)
- (0ad70fc) [Release 9.4.10.1](https://github.com/ai/autoprefixer-rails/commit/0ad70fcea2c70905e61af702a91ab0b07153f118)


</details>